swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -167,6 +171,15 @@ public final class Store: ObservableObject {
     }
   }
 
+  func toggleResource(resource: String, enabled: Bool) {
```

Comment:
```suggestion
  func toggleResourceDisabled(resource: String, enabled: Bool) {
```

There will also be a `toggleResourceFavorite` or similar, maybe not in this file, but somewhere. And although "toggle Resource enabled" is closer to "toggle Resource" in meaning, this would emphasize that it's about enabling, not about toggling, since favorites also toggle